examples/remote_control_cozmo.py

```diff
@@ -32,11 +32,41 @@
     sys.exit("Cannot import from flask: Do `pip3 install --user flask` to install")
 
 try:
-    from PIL import Image
+    from PIL import Image, ImageDraw
 except ImportError:
     sys.exit("Cannot import from PIL: Do `pip3 install --user Pillow` to install")
 
 
+# Annotator for displaying RobotState (position, etc.) on top of the camera feed
+class RobotStateDisplay(cozmo.annotate.Annotator):
+    def apply(self, image, scale):
+        d = ImageDraw.Draw(image)
+
+        bounds = [3, 0, image.width, image.height]
+
+        def print_line(text_line):
+            nonlocal bounds
```

Comment:
you don't technically need this `nonlocal` statement here as you're only referencing an existing variable, not assigning to it.. ie.. you'd need if it you'd done `bound=.. new bounds`, but as it is you're only mutating a value within an existing array, which as it doesn't exist locally will find by the usual scope rules